package chap10;

public class ExceptionEx5 {
	public static void main(String[] args) throws RuntimeException {
		try {
			first();
		}catch(Exception e) {
			System.out.println("숫자만 가능합니다.");
			e.printStackTrace();
		}
	}

	private static void first()throws RuntimeException {
		System.out.println("first 메소드");
		second();
	}

	private static void second()throws RuntimeException {
		System.out.println("second 메소드");
		System.out.println(Integer.parseInt("abc"));
		
	}

}
